### PR TITLE
Moves the account app's migrations to the access app, allowing them to run.

### PR DESCRIPTION
**Pull request by chuckharmston**
r?

### CODE REVIEW

File: mkt/access/migrations/0003_add_contenttools_groups.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 from django.db import migrations
+from django.db.models.loading import get_model
 
 
 def create_addon_groups(apps, schema_editor):
-    Group = apps.get_model('Access', 'Group')
```
Comment (diox) [inline mkt/access/migrations/0003_add_contenttools_groups.py]: lowercase `access`, it's an app name.

Comment (chuckharmston) [inline mkt/access/migrations/0003_add_contenttools_groups.py]: It's actually not an app at all in the Django 1.7 sense, hence tests failing and the below line being added.

Comment (diox) [inline mkt/access/migrations/0003_add_contenttools_groups.py]: and do use `apps.get_model`, never use `django.db.models.loading` in a migration, you might end up with a wrong version of the model.

Comment (chuckharmston) [inline mkt/access/migrations/0003_add_contenttools_groups.py]: You can't use apps.get_model unless you have an AppConfig object, right?

Comment (chuckharmston) [inline mkt/access/migrations/0003_add_contenttools_groups.py]: Okay, I give up. I *definitely* got an exception that said something along the lines of "no AppConfig with that label exists" earlier, but it seems to be working now.

`¯\_(ツ)_/¯`

**Comment by ngokevin** [comment]
r+

**Pull request merged**